src/components/calcite-input/calcite-input.tsx

```diff
@@ -447,7 +447,13 @@ export class CalciteInput {
   };
 
   private setDisabledAction(): void {
-    if (this.slottedActionEl) (this.slottedActionEl as HTMLElement).setAttribute("disabled", "");
+    if (!this.slottedActionEl) {
+      return;
+    }
+
+    this.disabled
+      ? ((this.slottedActionEl as HTMLElement) as HTMLElement).setAttribute("disabled", "")
+      : ((this.slottedActionEl as HTMLElement) as HTMLElement).removeAttribute("disabled");
```

Comment:
Doesn't Stencil take care of adding/removing the `disabled` attribute for us if this is reflected?

Reply:
This isn't on the host element.

Reply:
Woops!  Totally get it now :)